packages/relayer/src/provider-relayer.ts

```diff
@@ -1,16 +1,24 @@
 import { TransactionResponse, BlockTag, Provider } from '@ethersproject/providers'
-import { ethers } from 'ethers'
+import { ethers, providers } from 'ethers'
 import { walletContracts } from '@0xsequence/abi'
-import { SignedTransactions, Transaction } from '@0xsequence/transactions'
+import { computeMetaTxnHash, SignedTransactions, Transaction } from '@0xsequence/transactions'
 import { WalletContext } from '@0xsequence/network'
 import { WalletConfig, addressOf } from '@0xsequence/config'
 import { BaseRelayer, BaseRelayerOptions } from './base-relayer'
 import { FeeOption, Relayer } from '.'
+import { Optionals, Mask } from '@0xsequence/utils'
 
 const DEFAULT_GAS_LIMIT = ethers.BigNumber.from(800000)
 
 export interface ProviderRelayerOptions extends BaseRelayerOptions {
-  provider: Provider
+  provider: Provider,
+  waitPoolRate?: number,
+  deltaBlocksLog?: number
```

Comment:
whats `deltaBlocksLog`? doesnt seem we're using it either anywhere else in the code

Reply:
That should be used instead of the hardcoded values, I fixed it, thanks!